### DISCUSSION TITLE
Android TV notification with images

### DISCUSSION THREAD

**Discussion by fredamn76** [blakeblackshear/frigate#3557] [Ask A Question] [answered]
Hi. I'm trying to get Frigate images (after detection) to show on my Android TV. I can't get it to show a image. Does anyone have a solution?

**Comment by fredamn76** [comment]
I solved it 😊.

**Reply by chaospup** [reply]
@fredamn76 are you running Frigate in a separate container container than HA or are you running Frigate within HA?

**Comment by fredamn76** [comment, marked as answer]
If someone is interested this is my solution.

> service: notify.shield_android_tv
data:
  title: Information
  message: A {{trigger.payload_json['after']['label']}} has entered the yard.
  data:
    duration: 5
    position: top-right
    fontsize: medium
    transparency: 75%
    color: black
    interrupt: 0
    image:
      url: https://HA-URL/api/frigate/notifications/{{trigger.payload_json['after']['id']}}/thumbnail.jpg

>

**Reply by MivraMe** [reply]
Hi, I was checking your automation but I don't understand what is your trigger

**Reply by NickM-27** [reply]
trigger would be mqtt frigate/events topic

**Comment by jubjubrsx** [comment]
are you saying you just installed "notifications for androidtv" on your new googletv?

**Reply by MivraMe** [reply]
Yes I do that and it works great

**Comment by fredamn76** [comment]
I'm running it in a separate container

Vänligen
Fredrik Fällman

Den sön 18 feb. 2024 15:32chaospup ***@***.***> skrev:
